sdc/tests/test_rolling.py

```diff
@@ -400,107 +401,164 @@ def test_impl2(df, df2, w, c):
             pd.testing.assert_frame_equal(hpat_func(*args), test_impl2(*args))
             pd.testing.assert_frame_equal(hpat_func(*args), test_impl2(*args))
 
-    @skip_sdc_jit('Series.rolling.min() unsupported exceptions')
-    def test_series_rolling_unsupported_values(self):
-        def test_impl(series, window, min_periods, center,
+    def _get_assert_equal(self, obj):
```

Comment:
This looks like a very handy function. Should we maybe move it to test_utils and use it all other tests as well?